### PR TITLE
core: Remove non-numeric value warning in data table size control

### PR DESCRIPTION
**Pull request by javierag**
This warning was triggered when you selected 'all rows' in
the data table size control

### CODE REVIEW

File: main/core/src/EBox/Model/DataTable.pm [hunk 1]
```diff
@@ -2649,7 +2649,7 @@ sub setPageSize
         throw EBox::Exceptions::MissingArgument("Missing field rows");
     }
 
-    if ($rows < 0) {
+    if (($rows ne '_all') and ($rows < 0)) {
```
Comment (sixstone-qq) [inline main/core/src/EBox/Model/DataTable.pm]: I miss this value in the doc.

**Comment by sixstone-qq** [comment]
:+1:

File: main/core/src/EBox/Model/DataTable.pm (hunk 1 above)
Comment (javierag) [inline main/core/src/EBox/Model/DataTable.pm]: Fixed

**Comment by jacalvo** [comment]
Missing changelog entry.

**Pull request merged**